### PR TITLE
Feature/scopes matching

### PR DESCRIPTION
**Pull request by aleksandra-bel**
## Description
This is implementation of Mapping scopes to credentials
Closes: https://github.com/eclipse-tractusx/managed-identity-wallet/issues/130

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: src/main/java/org/eclipse/tractusx/managedidentitywallets/exception/MissingVcTypesException.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ *  under the License.
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ * ******************************************************************************
+ */
+
+package org.eclipse.tractusx.managedidentitywallets.exception;
+
+public class MissingVcTypesException extends RuntimeException {
+
```
Comment (borisrizov-zf) [inline src/main/java/org/eclipse/tractusx/managedidentitywallets/exception/MissingVcTypesException.java:25]: Missing a `serialVersionUid = 1L`.

File: src/main/java/org/eclipse/tractusx/managedidentitywallets/exception/PermissionViolationException.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ *  under the License.
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ * ******************************************************************************
+ */
+
+package org.eclipse.tractusx.managedidentitywallets.exception;
+
+public class PermissionViolationException extends RuntimeException {
+
```
Comment (borisrizov-zf) [inline src/main/java/org/eclipse/tractusx/managedidentitywallets/exception/PermissionViolationException.java:25]: Missing `serialVersionUID = 1L`

**Review by borisrizov-zf — COMMENTED** [review]
Looks good so far. I think it's safe to merge after you've pushed the tests.

File: src/main/java/org/eclipse/tractusx/managedidentitywallets/exception/PermissionViolationException.java (hunk 2 above)
Comment (aleksandra-bel) [reply src/main/java/org/eclipse/tractusx/managedidentitywallets/exception/PermissionViolationException.java:29]: added in [2f2e498](https://github.com/eclipse-tractusx/managed-identity-wallet/pull/259/commits/2f2e4988724e5af6dd45a3281ba4e5b5244660a6)

File: src/main/java/org/eclipse/tractusx/managedidentitywallets/exception/MissingVcTypesException.java (hunk 1 above)
Comment (aleksandra-bel) [reply src/main/java/org/eclipse/tractusx/managedidentitywallets/exception/MissingVcTypesException.java:29]: added in [2f2e498](https://github.com/eclipse-tractusx/managed-identity-wallet/pull/259/commits/2f2e4988724e5af6dd45a3281ba4e5b5244660a6)

**Review by borisrizov-zf — APPROVED** [review]

**Pull request merged**